### DISCUSSION TITLE
Will FreeTube ever come to android?

### DISCUSSION THREAD

**Discussion by Blu-byte** [FreeTubeApp/FreeTube#2120] [Q&A]
With the fall of Vanced or originally known as YouTube Vanced and many users having to go to NewPipe while others stay on the current version of Vanced till Google changes the API which breaks the Vanced client. I've seen a couple of users questioning if FreeTube will make a Android version of FreeTube.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
No. For full info please read our wiki https://docs.freetubeapp.io/faq/#do-you-plan-on-porting-freetube-to-other-platforms-android-ios-etc

**Comment by MarmadileManteater** [comment]
I hate to necropost, but a [fork of Freetube](https://github.com/MarmadileManteater/FreeTubeCordova) exists which solves this exact problem. It ports FreeTube to Cordova. [Cordova](https://cordova.apache.org/) is a very similar concept to Electron, but it exports to Android and iOS. Honestly, this fork is not perfect, but it demonstrates that the idea of putting FreeTube on Android or iOS is not as crazy of a concept as it might seem.

**Reply by PrestonN** [reply]
Just want to say that I love the work you've done on this so far. Hoping that once some issues get fixed that you'd be interested in sending these changes upstream? I'd love to start including these at some point. If there's anything you'd like for us to do on our end for this then please let me know.
